src/index.js

```diff
@@ -50,3 +50,4 @@ export {default as intersection} from "./intersection.js";
 export {default as subset} from "./subset.js";
 export {default as superset} from "./superset.js";
 export {default as union} from "./union.js";
+export {InternMap, InternSet} from "internmap";
```

Comment:
Why export these here?

Is the intention for these to end up as `d3.InternMap` and `d3.InternSet` (via [`export * from "d3-array";`](https://github.com/d3/d3/blob/master/index.js#L2))?

Reply:
Yes.